src/GUI/src/CommandHandler.cpp

```diff
@@ -91,7 +91,7 @@ namespace GUI {
             std::string teamName;
 
             if (!(ss >> cmd >> id >> x >> y >> orientation >> level >> teamName)
-                || orientation < 0 || orientation > 3) {
+                || orientation < 1 || orientation > 4) {
```

Comment:
dafuk is 1 ? and 4 ?

Reply:
```suggestion
                || orientation < UP || orientation > LEFT) {
```
on peut utiliser les enum comme des macro